commands/utils/set_args.js

```diff
@@ -434,6 +434,10 @@ function sync_args_from_cmd(args) {
       lt_config["run_settings"]["npm_via_tunnel"] = false;
     }
 
+    if ("max_duration" in args) {
+      lt_config["run_settings"]["max_duration"] = parseFloat(args["max_duration"]);
```

Comment:
duration should be int

Reply:
it is kept float intentional, as we want float number also. we are converting it to floor value in liisa.

Reply:
Can you also keep this as int